conf/license.php

```diff
@@ -4,33 +4,31 @@
  * wiki contents under. Do not change this file, but create a
  * license.local.php instead.
  */
-
 $license['cc-zero'] = array(
     'name' => 'CC0 1.0 Universal',
-    'url'  => 'http://creativecommons.org/publicdomain/zero/1.0/',
+    'url'  => 'http://creativecommons.org/publicdomain/zero/1.0/deed.'.$conf['lang'],
 );
 $license['publicdomain'] = array(
     'name' => 'Public Domain',
-    'url'  => 'http://creativecommons.org/licenses/publicdomain/',
+    'url'  => 'http://creativecommons.org/licenses/publicdomain/deed.'.$conf['lang'],
 );
 $license['cc-by'] = array(
     'name' => 'CC Attribution 4.0 International',
-    'url'  => 'http://creativecommons.org/licenses/by/4.0/',
+    'url'  => 'http://creativecommons.org/licenses/by/4.0/deed.'.$conf['lang'],
 );
 $license['cc-by-sa'] = array(
     'name' => 'CC Attribution-Share Alike 4.0 International',
-    'url'  => 'http://creativecommons.org/licenses/by-sa/4.0/',
+    'url'  => 'http://creativecommons.org/licenses/by-sa/4.0/deed.'.$conf['lang'],
 );
 $license['gnufdl'] = array(
     'name' => 'GNU Free Documentation License 1.3',
-    'url'  => 'http://www.gnu.org/licenses/fdl-1.3.html',
+    'url'  => 'http://www.gnu.org/licenses/fdl-1.3.'.$conf['lang'].'.html',
```

Comment:
Since this is kind of a "legal document" I don't think having the possibility of pointing it to a non-existing webpage is a good idea.

Reply:
I mean to add more language pages on the license sites :)

Reply:
I agree, but if they are not ready, I would prefer not to take the risk of pointing a legal link to a 404 webpage.

Reply:
Rewrite to #2967